src/js/thirdparty/ws.js

```diff
@@ -184,16 +186,11 @@ class BunWebSocket extends EventEmitter {
 
     if (typeof data === "number") data = data.toString();
 
-    // deviation: we don't support ping
-    emitWarning("ping()", "ws.WebSocket.ping() is not implemented in bun");
+    this.#ws.ping(data);
```

Comment:
does this need a try/catch to call the callback with an error? will it ever throw?